config/defaults.mk

```diff
@@ -244,6 +245,10 @@ GECKO_DIR = @MFEM_DIR@/../gecko
 GECKO_OPT = -I$(GECKO_DIR)/inc
 GECKO_LIB = -L$(GECKO_DIR)/lib -lgecko
 
+# Ginkgo library configuration (currently not needed)
+Ginkgo_OPT =
+Ginkgo_LIB =
```

Comment:
These should be defined based on `GINKGO_DIR` -- the first variable gives the options needed for compiling (e.g. `-I$(GINKGO_DIR)/include`) and the second one -- the options for linking (e.g. `$(XLINKER)-rpath,$(GINKGO_DIR)/lib -L$(GINKGO_DIR)/lib -lginkgo`).

The variable `GINKGO_DIR` can be defined similar to other `*_DIR` variables, e.g.
```makefile
GINKGO_DIR = @MFEM_DIR@/../ginkgo/install
```

Reply:
Also, the names should be capitalized: `GINKGO_OPT` and `GINKGO_LIB` -- this is expected by the main makefile where the library is spelled `GINKGO`.